tests/app/config/packages/neusta_converter.yaml

```diff
@@ -0,0 +1,16 @@
+neusta_converter:
+  converter:
+    test.person.converter:
+      converter: Neusta\ConverterBundle\Converter\DefaultConverter
```

Comment:
if we can use a Symfony symbolic name here I would prefer instead of FQCN.

Reply:
What do you mean by "symbolic name"?

Reply:
Something like neusta.default_converter which is a Symfony label

Reply:
First: the `converter` key is optional if you want to use the `DefaultConverter`, so you can omit it.

Second: the value is a service name, so you can use whatever you want. If you register your custom converter as "my_fancy_converter" instead of using the FQCN you can use that name.